src/core/function_subscription_migrations.tf

```diff
@@ -0,0 +1,92 @@
+module "function_subscriptionmigrations_snet" {
+  source               = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v2.1.0"
+  name                 = format("%s-fn-sub-migrations-snet", local.project)
+  address_prefixes     = var.cidr_subnet_selfcare_be
+  resource_group_name  = data.azurerm_resource_group.vnet_common_rg.name
+  virtual_network_name = data.azurerm_virtual_network.vnet_common.name
+  service_endpoints = [
+    "Microsoft.Storage",
+    "Microsoft.Web",
+  ]
+
+  delegation = {
+    name = "default"
+    service_delegation = {
+      name    = "Microsoft.Web/serverFarms"
+      actions = ["Microsoft.Network/virtualNetworks/subnets/action"]
+    }
+  }
+}
```

Comment:
when you share app service plans u need to reuse associated subnet, so u need to remove this subnet

Reply:
Tried to fix with this: https://github.com/pagopa/io-infra/pull/77/commits/92520c88d225ef06b2540812477ad81d97c28b09